extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java

```diff
@@ -204,11 +205,16 @@ public void build(RecorderContext recorderContext, HibernateOrmRecorder recorder
             return;
         }
 
+        // we only support the default datasource for now
```

Comment:
I think we should at least warn users if more than one is configured

Reply:
Why? They can be used for other purposes. And this patch doesn't change that, it just changes how the default is propagated.

Reply:
I am just saying that a user has no way of knowing this and as someone who looked at it for the first time, it seems suprising to make such a assumption without any warning or anything.
You know this part way better than almost anyone, so I'll leave it to you to decide

Reply:
It has been like that since forever so let's not discuss this here. We have an issue for multiple PU support and a lot of people complaining there so I think users are aware of that :).

Reply:
Fine by my, since I'm not the one that will be listening to those users complain :P